code/modules/cargo/packs/goodies.dm

```diff
@@ -0,0 +1,77 @@
+
+/datum/supply_pack/goody
+	access = NONE
+	group = "Goodies"
+	goody = PACK_GOODY_PRIVATE
+
+/datum/supply_pack/goody/combatknives_single
+	name = "Combat Knife Single-Pack"
+	desc = "Contains one sharpened combat knive. Guaranteed to fit snugly inside any Nanotrasen-standard boot."
+	cost = 800
+	contains = list(/obj/item/kitchen/knife/combat)
+
+/datum/supply_pack/goody/sologamermitts
+	name = "Insulated Gloves Single-Pack"
+	desc = "The backbone of modern society. Barely ever ordered for actual engineering."
+	cost = 800
+	contains = list(/obj/item/clothing/gloves/color/yellow)
+
+/datum/supply_pack/goody/firstaidbruises_single
+	name = "Bruise Treatment Kit Single-Pack"
+	desc = "A single brute first-aid kit, perfect for recovering from being crushed in an airlock. Did you know people get crushed in airlocks all the time? Interesting..."
+	cost = 330
+	contains = list(/obj/item/storage/firstaid/brute)
+
+/datum/supply_pack/goody/firstaidburns_single
```

Comment:
Why cant cargo buy any more firstaid packs of any type? Thats really limiting on what cargo can restock : /

Reply:
medbay is stuffed full of medikits anyway. Why care about a non-issue?